main.go

```diff
@@ -59,10 +56,6 @@ func generateServiceEndpoint(user, pass, path string) string {
 }
 
 func init() {
-	gearmanAdminUser := getEnv("GEARMAN_ADMIN_USER")
```

Comment:
Can you remove these from the yml and make a corresponding `ark-config` PR to remove them? 🎉